src/services/paras/ParasService.spec.ts

```diff
@@ -485,9 +485,9 @@ describe('ParasService', () => {
 
 			const expectedResponse = {
 				at: expectedAt,
-				beginEnd: '1000',
-				finishEnd: '21000',
-				phase: 'vrfDelay',
+				beginEnd: '780000',
+				finishEnd: '800000',
+				phase: 'endPeriod',
```

Comment:
Would be good to have a test for vrfDelay, startPeriod, endPeriod, and auction over

Reply:
We were already testing for `endPeriod`, and when there is no current auctionInfo available. I added vrfDelay, and startPeriod and cleaned it all up so it makes sense together.